pkg/kubelet/dockertools/manager_test.go

```diff
@@ -30,7 +30,6 @@ import (
 	"time"
 
 	dockertypes "github.com/docker/engine-api/types"
-	containertypes "github.com/docker/engine-api/types/container"
```

Comment:
In the code base, there is only one imported https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockertools/manager_test.go#L33.

I suppose this should be fixed after rebasing? :)